server/arm.py

```
@@ -103,21 +121,57 @@ def invalidate_arm_connection(serial_ports, reason="unknown"):
             f"encode:{joint_name}",
-            f"[ARM DEBUG] {joint_name} raw={value}, scaled={scaled}, "
-            f"hex={scaled.to_bytes(2, 'big', signed=True).hex()}",
+            f"[ARM DEBUG] {joint_name} val={value} frac={fraction:.4f} "
+            f"mant={mantissa} wire={exp_hex}{mantissa_hex}",
         )
-    except Exception:
-        scaled = 0
+        return exp_hex + mantissa_hex
 
-    return scaled.to_bytes(2, "big", signed=True).hex()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)